habitLoggerApp.Natephor/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using HabitLoggerApp;
+
+var userInterface = new UserInterface();
+var habitManager = new HabitManager();
+
+var appIsRunning = true;
+while (appIsRunning)
+{
+    userInterface.DisplayMenu(true);
+
+    while (appIsRunning)
+    {
+        var input = userInterface.GetUserInput("Enter your choice: ");
+        switch (input)
+        {
+            case "1":
+                var habits = habitManager.GetAllHabits();
+                userInterface.DisplayHabits(habits);
+                break;
+            case "2":
+                var body = userInterface.GetUserInput("Enter habit body: ");
+                var parsedDate = userInterface.GetUserDateInput("Enter habit date: ");
+                var quantity = userInterface.GetUserNumberInput("Enter quantity: ");
+
+                habitManager.CreateHabit(parsedDate, body, quantity);
+                break;
+            case "3":
+                var updatingId = userInterface.GetUserNumberInput("Enter the id to be updated: ");
```

Comment:
🟠 **No ID Validation**

💡 You have not validated that this ID exists. Meaning that the rest of this statement block needlessly executes.

![image](https://github.com/user-attachments/assets/fde08a50-696c-4b84-a295-f437e4a07826)